feature-libs/user/account/components/guards/login-as-guest.guard.ts

```diff
@@ -23,6 +23,8 @@ export class LoginAsGuestGuard {
   protected router = inject(Router);
   protected semanticPathService = inject(SemanticPathService);
 
+  routeName: string = 'loginForm';
```

Comment:
```suggestion
  protected routeName: string = 'loginForm';
```

Reply:
`medium:` let's change this to protected method for better customisability.

Reply:
done